### DISCUSSION TITLE
Accessing users email with JS SDK

### DISCUSSION THREAD

**Discussion by dimitarnikolovv** [pocketbase/pocketbase#4194] [Q&A] [answered]
Before I was able to access the current logged user's email using the `authStore.model.email `. But after I updated from 0.20.2 to 0.20.5 I no longer can access the user's email that way. At least not on all auth records, which I have no idea why. I looked through the update logs but I can not find any changes regarding the authStore model. Keep in mind that I have not changed any settings or the `emailVisibility` field, which has been always set to `false`.

**Comment by ganigeorgiev** [comment]
There are no changes to the `emailVisibility` handling in the recent PocketBase or JS SDK releases.

Note that the user email is visible **only** by:
- admins
- the owner of the auth record
- by other users IF `emailVisibility` is `true`

If you use Next.js, make sure that there is no cache involved.

If you think that it is a PocketBase issue, please provide some details about your project (eg. how you are initializing the SDK) and ideally some reproducible steps and I'll try to investigate it.

**Reply by ganigeorgiev** [reply, marked as answer]
_Just for info in case someone stumble here - another user reported similar issue in [#2173](https://github.com/pocketbase/pocketbase/issues/2173#issuecomment-1932491484) and I was able to reproduce it with the provided collections export. The problem should be fixed with PocketBase [v0.21.2 release](https://github.com/pocketbase/pocketbase/releases/tag/v0.21.2)._
